src/styles/theme.scss

```diff
@@ -2,27 +2,27 @@
   font-family: "Lato";
   font-style: normal;
   font-weight: normal;
-  src: url('./fonts/lato-regular.eot'); /* IE9 */
-  src: url('./fonts/lato-regular.woff') format('woff'), /* Modern Browsers */
-       url('./fonts/lato-regular.ttf') format('truetype'); /* Mobile */
+  src: url("./fonts/lato-regular.eot"); /* IE9 */
+  src: url("./fonts/lato-regular.woff") format("woff"),
+    /* Modern Browsers */ url("./fonts/lato-regular.ttf") format("truetype"); /* Mobile */
 }
 
 @font-face {
   font-family: "Lato";
   font-style: normal;
   font-weight: bold;
-  src: url('./fonts/lato-bold.eot'); /* IE9 */
-  src: url('./fonts/lato-bold.woff') format('woff'), /* Modern Browsers */
-       url('./fonts/lato-bold.ttf') format('truetype'); /* Mobile */
+  src: url("./fonts/lato-bold.eot"); /* IE9 */
+  src: url("./fonts/lato-bold.woff") format("woff"),
+    /* Modern Browsers */ url("./fonts/lato-bold.ttf") format("truetype"); /* Mobile */
```

Comment:
And here

Reply:
The comment should be at the end of the line, it's referring to the woff and woff2 formats.